EIPS/eip-9999.md

```diff
@@ -0,0 +1,166 @@
+---
+eip: 9999
```

Comment:
```suggestion
eip: 8071
```
Assigning next sequential EIP/ERC/RIP number.
Numbers are assigned by editors & associates.

Please also update the filename.